packages/ndla-ui/src/LearningPaths/LearningPathSticky.tsx

```diff
@@ -62,6 +62,7 @@ const SafeLinkCSS = css`
   ${mq.range({ until: breakpoints.tablet })} {
     height: ${FOOTER_HEIGHT_MOBILE};
     width: ${FOOTER_HEIGHT_MOBILE};
+    min-width: ${FOOTER_HEIGHT_MOBILE};
```

Comment:
Ser det er brukt rett over her, men litt rart at footer _height_ brukes på bredde?

Reply:
hm ja, var bare for å få den like bred som den var høy, men blir jo litt forvirrende naming kanskje...

Reply:
Ikkje så farilig. Berre å merge.